dotcom-rendering/src/components/QAndAExplainer.tsx

```diff
@@ -51,33 +51,35 @@ export const QAndAExplainer = ({
 }: Props) => {
 	return (
 		<>
```

Comment:
If you're wrapping everything in a `div`, then you no longer need the [JSX fragment](https://react.dev/reference/react/Fragment). Could you please update so we have no unnecessary wrapper?